client/src/pages/FriendsPolls.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import DashboardLayout from "../components/DashboardLayout";
+import FriendsPollsPage from "../components/FriendsPollsPage/FriendsPollsPage";
+import { NAVBAR_HEIGHT } from "../utils/constants";
+
+export default function Friends() {
```

Comment:
Function Friends already exist...
Better to match the function name and files name